templates/include/telemetry/environment.1.schema.json

```diff
@@ -5,7 +5,7 @@
       "properties": {
         "activeAddons": {
           "type": "object",
-          "additionalProperties": {
+          "properties": {
```

Comment:
I don't think this is right - this should be an `additionalProperties` field, but we should remove the "addon" piece. It should just be something like:
```
"additionalProperties": {
  "type": "object",
  "properties": { ... all the properties of an addon ...}
}
```